### DISCUSSION TITLE
Toggleable optimized ticklist

### DISCUSSION THREAD

**Discussion by Nervaq** [PurpurMC/Purpur#503] [__Dismissed__] [answered]
I would like to suggest to reimplement use-optimized-ticklist in config. Since i don't code nor any of our Mod on server. With this feature i believe piston tapes and TNT duping would be reliable just like in Vanilla.

**Comment by Apprisco** [comment, marked as answer]
tbh tnt duping is already reliable AF on paper and purpur... at least in my experience. Same with 0 tick smart pistons.

**Reply by Nervaq** [reply]
If by reliable you mean TNT dupers break when you unload chunks and piston tapes get stuck if there is not a consistent flow of blocks. Not to mention that the only thing how to reliably unstuck piston tapes doesnt work since comparators do not send updates like they should in vanilla.
